utils/utils.go

```diff
@@ -42,3 +42,14 @@ func GetEnv(env, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func GetEnvBoolWithDefault(envName string, def bool) bool {
```

Comment:
nit: doesn't `GetBoolAsStringEnvVar` already do what you need?

Reply:
Right..overlooked the function. Will remove this..